specTemplateARM.spec

```diff
@@ -0,0 +1,148 @@
+%define _package __package__
+%define _packagename __packagename__
+%define _version __version__
+%define _release __release__
+%define _prefix  __prefix__
+%define _sources_dir __sources_dir__
+%define _tmppath /tmp
+%define _packagedir __packagedir__
+%define _os __os__
+%define _platform __platform__
+%define _project __project__
+%define _author __author__
+%define _summary __summary__
+%define _url __url__
+%define _buildarch __buildarch__
+#%define _includedirs __includedirs__
+
+%define _unpackaged_files_terminate_build 0
+
+#
+# Binary RPM specified attributed (lib and bin)
+#
+Name: %{_packagename}
+Summary: %{_summary}
+Version: %{_version}
+Release: %{_release}
+Packager: %{_author}
+#BuildArch: %{_buildarch}
+License: __license_
+# Group: Applications/extern
+URL: %{_url}
+BuildRoot: %{_tmppath}/%{_packagename}-%{_version}-%{_release}-buildroot
+Prefix: %{_prefix}
+Requires: __requires_list__
+BuildRequires: __build_requires_list__
+AutoReqProv: no
```

Comment:
Probably *only* need `AutoReq: no` here

Reply:
no, we need the other lines to handle our own dependencies